bolt/lib/Core/DebugData.cpp

```diff
@@ -393,6 +393,7 @@ void DebugARangesSectionWriter::writeARangesSection(
 DebugAddrWriter::DebugAddrWriter(BinaryContext *BC) : BC(BC) {
   Buffer = std::make_unique<AddressSectionBuffer>();
   AddressStream = std::make_unique<raw_svector_ostream>(*Buffer);
+  Map = AddressForDWOCU();
```

Comment:
I don't believe this is necessary. It's a member of your class. Default constructor will already be invoked when class instantiated.